### PR TITLE
Docker keycloak

### PR DESCRIPTION
**Pull request by ElisyanBot**
added basic keycloak img to the dock compose file

**When this is merged security will be activated**

### CODE REVIEW

**Comment by CosmicJay2k** [comment]
What about adding the realm import and file to this pull request?

File: SetupCompose.yml [hunk 1]
```diff
@@ -10,4 +10,15 @@ services:
...
+    image: quay.io/keycloak/keycloak:20.0.2
+    environment:
+     - KEYCLOAK_ADMIN=${KEYCLOAK_ADMIN_NAME}
+     - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}
+    command:
+     start-dev --import-realm&&--db=mysql --features=token-exchange&&db-url="http://localhost:3306" --db-username=${SPRING_DATASOURCE_USER} --db-password=${SPRING_DATASOURCE_PASSWORD}
+    volumes: 
+      -  ./keycloak/realms:/opt/keycloak/data/import
+    ports:
+      - "8000:8080"
```
Comment (CosmicJay2k) [inline SetupCompose.yml:24]: We should really decide on a port :D

Comment (CosmicJay2k) [reply SetupCompose.yml:24]: Decided! ^^ 8000

**Review by CosmicJay2k — APPROVED** [review]
I think we have everything we want in this pull now.

**Review by FredricNordin — APPROVED** [review]

**Pull request merged**